R/modules.R

```diff
@@ -23,15 +23,16 @@
 #' @param server (`function`) `shiny` module with following arguments:
 #'  - `id` - `teal` will set proper `shiny` namespace for this module (see [shiny::moduleServer()]).
 #'  - `input`, `output`, `session` - (not recommended) then [shiny::callModule()] will be used to call a module.
-#'  - `data` (optional) module will receive a `teal_data` object, a list of reactive (filtered) data specified in
-#'     the `filters` argument.
-#'  - `datasets` (optional) module will receive `FilteredData`. (See [`teal.slice::FilteredData`]).
-#'  - `reporter` (optional) module will receive `Reporter`. (See [`teal.reporter::Reporter`]).
-#'  - `filter_panel_api` (optional) module will receive `FilterPanelAPI`. (See [`teal.slice::FilterPanelAPI`]).
-#'  - `...` (optional) `server_args` elements will be passed to the module named argument or to the `...`.
+#'  - `data` (optional) If present, module will receive a `teal_data` object, a list of reactive (filtered)
+#'     data specified in the `filters` argument.
+#'  - `datasets` (optional) If present, module will receive `FilteredData`. (See [`teal.slice::FilteredData`]).
+#'  - `reporter` (optional) If present, module will receive `Reporter`. (See [`teal.reporter::Reporter`]).
+#'  - `filter_panel_api` (optional) If present, module will receive `FilterPanelAPI`.
+#'     (See [`teal.slice::FilterPanelAPI`]).
+#'  - `...` (optional) If present, `server_args` elements will be passed to the module named argument or to the `...`.
 #' @param ui (`function`) `shiny` UI module function with following arguments:
 #'  - `id` - `teal` will set proper `shiny` namespace for this module.
-#'  - `...` (optional) `ui_args` elements will be passed to the module named argument or to the `...`.
+#'  - `...` (optional) If present, `ui_args` elements will be passed to the module named argument or to the `...`.
```

Comment:
@pawelru did you add `If present` just after the word `(optional)`? for every `optional` parameter?

Reply:
Yes. It's because the english doesn't sound well.
After a short break from this PR I think that the current version is not perfect either :P Currently it's more describing a _flag_ behaviour. Let me enhance it more.

Reply:
`when provided`?